l1-contracts/deploy-scripts/InitializeL2WethToken.s.sol

```diff
@@ -5,7 +5,7 @@ pragma solidity ^0.8.20;
 
 import {Script, console2 as console} from "forge-std/Script.sol";
 import {stdToml} from "forge-std/StdToml.sol";
-import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {ITransparentUpgradeableProxy} from "@openzeppelin/contracts-v4/proxy/transparent/TransparentUpgradeableProxy.sol";
```

Comment:
You should also change `.ts` scripts. 
- https://github.com/matter-labs/era-contracts/blob/zk-evm-652-migrate-oz-dependencies-to-5x/l1-contracts/scripts/initialize-l2-weth-token.ts#L23
- https://github.com/matter-labs/era-contracts/blob/zk-evm-652-migrate-oz-dependencies-to-5x/l1-contracts/scripts/setup-legacy-bridge-era.ts#L19
- https://github.com/matter-labs/era-contracts/blob/zk-evm-652-migrate-oz-dependencies-to-5x/l1-contracts/src.ts/hyperchain-upgrade.ts#L29
- https://github.com/matter-labs/era-contracts/blob/zk-evm-652-migrate-oz-dependencies-to-5x/l1-contracts/test/unit_tests/utils.ts#L19